react/components/form/ImageUploader/index.tsx

```diff
@@ -57,20 +74,16 @@ class ImageUploader extends Component<Props, State> {
     } = this.props
     const { error, isLoading } = this.state
 
-    const FieldTitle = () => (
-      <FormattedMessage id={title as string}>
-        {text => <span className="w-100 db mb3">{text}</span>}
-      </FormattedMessage>
-    )
-
     const backgroundImageStyle = {
       backgroundImage: `url("${value}")`,
     }
 
     if (value) {
       return (
         <Fragment>
-          <FieldTitle />
+          <FormattedMessage id={title as string}>
```

Comment:
I know that, although `title` may be `null`, the user wouldn't be able to reach this part of the code if it was, because the block wouldn't have shown up in the list. However, perhaps we should consider using a separate type for "`null`-proofed" schemas instead of typecasting variables; the same goes for `editor`'s `editTreePath`, for example. (We can discuss that later, though.)